policyengine_us/parameters/gov/states/mt/dhs/tanf/income/deductions/child_care/amount.yaml

```
@@ -0,0 +1,14 @@
+description: Montana deducts up to this amount of dependent care expenses per child from countable income under the Temporary Assistance for Needy Families program.
+
+metadata:
+  period: month
+  label: Montana TANF monthly earned income childcare deduction per child
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-USD
```

```suggestion
  unit: currency-USD
```

`single_amount` parameters are usually scales such as [this](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/irs/deductions/tuition_and_fees/joint.yaml) parameter 

For a simple value we can just define the unit in the metadata